posthog/settings/data_stores.py

```diff
@@ -100,10 +100,23 @@ def postgres_config(host: str) -> dict:
 read_host = os.getenv("POSTHOG_POSTGRES_READ_HOST")
 if read_host:
     DATABASES["replica"] = postgres_config(read_host)
+    DATABASE_ROUTERS = ["posthog.dbrouter.ReplicaRouter"]
 
 if JOB_QUEUE_GRAPHILE_URL:
     DATABASES["graphile"] = dj_database_url.config(default=JOB_QUEUE_GRAPHILE_URL, conn_max_age=600)
 
+
+# Opt-in to using the read replica
+# Models using this will likely see better query latency, and better performance.
+# Immediately reading after writing may not return consistent data if done in <100ms
+# Pass in model classnames that should use the read replica
+
+# Note: Regardless of settings, cursor usage will use the default DB unless otherwise specified.
+# Database routers route models!
+replica_opt_in = os.environ.get("READ_REPLICA_OPT_IN", "")
+READ_REPLICA_OPT_IN: list[str] = replica_opt_in.split(",")
```

Comment:
minor nit: `get_list` here would save some minor annoyances, and keeps it consistent with our other list based env vars.